### PR TITLE
Update README section for WSDL and Open API specs

### PR DESCRIPTION
**Pull request by velsietis**
Also added a Sandbox APIs section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 # NHS Wales API Catalogue
 
+The files in this repository are used to generate documentation and user guides for NHS Wales APIs.
```
Comment (Geoff1900) [inline README.md:3]: How about using an active voice throughout..._We use the files in this repository to generate documentation and user guides for NHS Wales APIs_.

File: README.md [hunk 2]
```diff
@@ -14,11 +18,19 @@
...
+
+## Contributing
+
+The catalogue is in the open and we encourage contributions. So if you come across a spelling mistake or wish to add an NHS API, feel free to send a [pull request](https://github.com/nwisbeta/api-catalogue/pulls) or you can open a [issue](https://github.com/nwisbeta/api-catalogue/issues) to report it to us.
+
+Read our [contributing guidelines](CONTRIBUTING.md) for more information.
+
+## Catalogue Structure
+
+The catalogue files are structured as shown below:
```
Comment (Geoff1900) [inline README.md:33]: Perhaps convert to an Active Voice?

File: README.md [hunk 3]
```diff
@@ -89,8 +101,36 @@ The status of an API is determined by availability.
...
+Standard formats such as [WSDL][WSDL] and [Open API][OpenAPI] provide consumers with enough information to start
+developing client applications, even without access to the actual API.  
+They also allow us to generate more detailed documentation in the HTML view of the catalogue.
+
+>**NOTE**  
+We recommend that you don't include the actual url/endpoint of your api servers, instead replace it with `https://private.url`  
+In the HTML view of the catalogue the URL will appear as `https://sandbox.api.nhs.wales/{system-id}/{api-id}/`
+
+#### Providing an Open API document
+For Open API documents, the description of each _path/operation_ is used to generate documentation in the HTML view of the cataloge.
```
Comment (Geoff1900) [inline README.md:115]: Again, perhaps use an active voice and fix typo [replace cataloge with catalogue]

**Review by Geoff1900 — CHANGES_REQUESTED** [review]
I suggest re-writing some of the sentences using an active voice throughout - see https://docs.microsoft.com/en-us/style-guide/grammar/verbs which I intend to become a standard writing style for docs in future.

File: README.md (hunk 3 above)
Comment (velsietis) [reply README.md:115]: Not sure about active voice as a blanket rule. I'd say we should select what most clearly communicates the key information.

With that in mind, consider these options:
1. Passive: _the description [...] is used_
2. Active: _we use the description..._
3. Active: _our catalogue processor uses the description_

In Option 1, I would say the passive voice hints at the existence of an automated process (i.e. no _active_ human involvement) without dwelling on what that process is.
Option 2 hints at a more manual process and slightly distracts from the key information we want to communicate here - the descriptions and how they're used, not about who or what acts upon them.
Option 3 is explicit but further distracts from the key point


```suggestion
For Open API documents, the description of each _path/operation_ is used to generate documentation in the HTML view of the catalogue.
```

**Review by Geoff1900 — APPROVED** [review]
Agreed.

**Pull request merged**